wasm-rpc-stubgen/src/wit.rs

```diff
@@ -309,10 +600,16 @@ pub fn get_package_name(wit: &Path) -> anyhow::Result<PackageName> {
     Ok(pkg.name)
 }
 
+#[allow(clippy::enum_variant_names)]
 pub enum WitAction {
     CopyDepDir {
         source_dir: PathBuf,
     },
+    CopyWitStr {
```

Comment:
This name is very confusing to me, I don't understand what does it mean to "copy a WIT **string**". 

I think what it does is it writes a WIT source to the given file so why not call it `WriteWit`

Reply:
Ok `WriteWit` makes sense